client/StatusTable/components/DayRow.js

```diff
@@ -0,0 +1,47 @@
+import React, {useEffect, useState} from "react";
+import * as moment from "moment";
+
+function DayRow(props) {
+    const [currentWorkTime, setCurrentWorkTime] = useState(0);
+    const { startSelected } = props;
+    const {
+        nameOfTheDay,
+        numberOfTheDay,
+        startTime,
+        finishTime,
+        fullTime,
+        realTime
+    } = props.workday;
+
+    useEffect(() => {
+        let timer_id;
+        if (startSelected) {
+            const currentDay = moment().date();
+            if (numberOfTheDay === currentDay){
+                timer_id = setInterval(() => {
+                    setCurrentWorkTime(cur => cur + 1);
+                }, 60*1000);
+            }
+        }
+        return () => {
+            setCurrentWorkTime(0);
+            clearInterval(timer_id);
+        };
+    }, [startSelected]);
+
+    // taken time + current count of minutes
+    const currentTime = moment(fullTime, "HH:mm").add(currentWorkTime, "minutes").format("H:mm");
```

Comment:
you can extract logic on calculating working time online to the separate component. Because there is no need to re-render all other cells every time.